charts/islandis/values.dev.yaml

```diff
@@ -189,6 +189,7 @@ api:
     ISLYKILL_CERT: '/etc/config/islyklar.p12'
     NODE_OPTIONS: '--max-old-space-size=464'
     PARTY_LETTER_REGISTRY_BASE_API_URL: 'http://web-party-letter-registry-api.party-letter-registry.svc.cluster.local'
+    PAYMENT_SCHEDULE_XROAD_PROVIDER_ID: 'IS-DEV/GOV/10021/FJS-Public'
```

Comment:
Where does all of these value changes come from? Can't see corresponding dsl changes. Also, should these start with XROAD_ to match sindri's recent naming convention?

Reply:
probably, i didn't even notice that those had changed. I thought we had a lint task for that